go_pprof_scraper/datadog_checks/go_pprof_scraper/data/conf.yaml.example

```diff
@@ -0,0 +1,82 @@
+## All options defined here are available to all instances.
+#
+init_config:
+
+    ## @param tags - list of strings - optional
+    ## A list of tags to attach to every metric and service check emitted by this integration.
+    ##
+    ## Learn more about tagging at https://docs.datadoghq.com/tagging
+    #
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
+
+## Every instance is scheduled independent of the others.
+#
+instances:
+
+    ## @param service_name - string - required
+    ## Name of the service, which will be tagged as "service:<SERVICE>" on uploaded profiles
+    #
+  - service_name: myservice
```

Comment:
If I call this `service`, the unit tests fail on the `assert_service_check` calls with an error saying that a `service` tag isn't allowed. I saw I can set a variable somewhere to make this error go away, but is that the appropriate thing to do?

Reply:
`service` is a reserved parameter. You should use the `default` template to get it and add your other parameters to the `spec.yaml` file. You can find an example [here](https://github.com/DataDog/integrations-core/blob/master/airflow/assets/configuration/spec.yaml), which generates [this file](https://github.com/DataDog/integrations-core/blob/master/airflow/datadog_checks/airflow/data/conf.yaml.example)

Reply:
As background, for profiling we always want to have a `service` tag for uploaded profiles, so I want to make it required for configuring this integration. In my tests I add the service value to the config. If I add `service` to the config, though, I get an error like this when I run the end-to-end tests:

```
E   Exception: Metric go_pprof_scraper.can_connect was submitted with a forbidden tag: service. Please rename this tag, or skip the tag validation with DDEV_SKIP_GENERIC_TAGS_CHECK environment variable.
```

I've changed the config to use the `default` template anyway. Should I just add the `DDEV_SKIP_GENERIC_TAGS_CHECK` environment variable like this error message says?

Reply:
I decided to skip the generic tags check.